letsencrypt/cli.py

```diff
@@ -841,9 +841,23 @@ def _plugins_parsing(helpful, plugins):
     helpful.add_plugin_args(plugins)
 
 
-def _setup_logging(args):
-    level = -args.verbose_count * 10
-    fmt = "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
+def setup_log_file_handler(args, logfile, fmt):
```

Comment:
Maybe some of these common functions should be moved to `log.py` or `le_util.py`.